manim/utils/tex_templates.py

```diff
@@ -0,0 +1,1022 @@
+epigrafica.add_to_preamble(
+    r"""
+\usepackage[LGR,OT1]{fontenc}
+\usepackage{epigrafica}
+\usepackage[basic,LGRgreek,defaultmathsizes]{mathastext}
+\let\varphi\phi
+\linespread{1.2}
+"""
+)
+
+
+# Libris ADF with Fourier
+librisadff = _new_ams_template()
+librisadff.description = "Libris ADF with Fourier"
+librisadff.add_to_preamble(
+    r"""
+\usepackage[T1]{fontenc}
+\usepackage[upright]{fourier}
+\usepackage{libris}
+\renewcommand{\familydefault}{\sfdefault}
+\usepackage[noasterisk]{mathastext}
+"""
+)
+
+
+# Venturis ADF with Fourier (Italic)
+italicvanturisadff = _new_ams_template()
+italicvanturisadff.description = "Venturis ADF with Fourier (Italic)"
+italicvanturisadff.add_to_preamble(
+    r"""
+\usepackage{fourier}
+\usepackage[lf]{venturis}
+\usepackage[italic,defaultmathsizes,noasterisk]{mathastext}
+"""
+)
+
+
+# GFS Bodoni
+gfsbodoni = _new_ams_template()
+gfsbodoni.description = "GFS Bodoni"
+gfsbodoni.add_to_preamble(
+    r"""
+\usepackage[T1]{fontenc}
+\renewcommand{\rmdefault}{bodoni}
+\usepackage[LGRgreek]{mathastext}
+\let\varphi\phi
+\linespread{1.06}
+"""
+)
+
+
+# BrushScriptX-Italic (PX math and Greek)
+brushscriptxpx = _new_ams_template()
+brushscriptxpx.description = "BrushScriptX-Italic (PX math and Greek)"
+brushscriptxpx.add_to_preamble(
+    r"""
+\usepackage[T1]{fontenc}
+\usepackage{pxfonts}
+%\usepackage{pbsi}
+\renewcommand{\rmdefault}{pbsi}
+\renewcommand{\mddefault}{xl}
+\renewcommand{\bfdefault}{xl}
+\usepackage[defaultmathsizes,noasterisk]{mathastext}
+"""
+)
+brushscriptxpx.add_to_document(
+    r"""\boldmath
+"""
+)
+brushscriptxpx.tex_compiler = "xelatex"
+brushscriptxpx.output_format = ".xdv"
+
+
+# URW Avant Garde (Symbol Greek)
+urwagsg = _new_ams_template()
+urwagsg.description = "URW Avant Garde (Symbol Greek)"
+urwagsg.add_to_preamble(
+    r"""
+\usepackage[T1]{fontenc}
+\usepackage{avant}
+\renewcommand{\familydefault}{\sfdefault}
+\usepackage[symbolgreek,defaultmathsizes]{mathastext}
+"""
+)
+
+
+# Times with Fourier (Italic)
+italictimesf = _new_ams_template()
+italictimesf.description = "Times with Fourier (Italic)"
+italictimesf.add_to_preamble(
+    r"""
+\usepackage{fourier}
+\renewcommand{\rmdefault}{ptm}
+\usepackage[italic,defaultmathsizes,noasterisk]{mathastext}
+"""
+)
+
+
+# Helvetica with Fourier (Italic)
+italichelveticaf = _new_ams_template()
+italichelveticaf.description = "Helvetica with Fourier (Italic)"
+italichelveticaf.add_to_preamble(
+    r"""
+\usepackage[T1]{fontenc}
+\usepackage[scaled]{helvet}
+\usepackage{fourier}
+\renewcommand{\rmdefault}{phv}
+\usepackage[italic,defaultmathsizes,noasterisk]{mathastext}
+"""
+)
+
+
+class TexFontTemplates(object):
+    """
+    A collection of TeX templates for the fonts described at http://jf.burnol.free.fr/showcase.html
+
+    These templates are specifically designed to allow you to typeset formulae and mathematics using
+    different fonts. They are based on the mathastext LaTeX package.
+
+    Usage
```

Comment:
also goes after Notes i believe
```suggestion
    Examples
```

Reply:
I meant usage as in "this is how it is used".
Namely (next line) "As a value for the keyword argument tex_template of Tex() and MathTex() mobjects"

And then I give a one line example just to show precisely how I meant that. (i.e. the syntax)

Calling this section 'examples' doesn't feel quite right to me.

Reply:
oh, this is just because there is no Usage section in the numpy format

Reply:
I'm sorry. I didn't know there even was such a thing as "numpy format".

I learned the other day that I should change my `#` comment documentation into `"""` comment documentation and that these comments should contain restructured text (rst).

Reply:
haha dont worry, it's alright! The guidelines should be helpful here then. Make sure to take a look!

Reply:
this still wasnt resolved; i guess we should just add this to the long description then?